addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -435,9 +435,8 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
             const fadeout = this.tours[tour_name].fadeout;
             core.bus.trigger('show-effect', {
                 type: "rainbow_man",
-                message,
+                message: owl.markup(message),
```

Comment:
grep rainbowManMessage and add markup in tour defs, where necessary + check missing _t

Reply:
remove markup here

Reply:
add markup l 433